activesupport/lib/active_support/encrypted_file.rb

```diff
@@ -74,6 +84,7 @@ def writing(contents)
 
 
       def encrypt(contents)
+        raise InvalidKeyLengthError if key&.length != self.class.expected_key_length
```

Comment:
How about extracting a method here? This line seems quite low level compared to the one below.

```patch
diff --git a/activesupport/lib/active_support/encrypted_file.rb b/activesupport/lib/active_support/encrypted_file.rb
index f2923224a1..43b1a4f467 100644
--- a/activesupport/lib/active_support/encrypted_file.rb
+++ b/activesupport/lib/active_support/encrypted_file.rb
@@ -84,7 +81,7 @@ def writing(contents)


       def encrypt(contents)
-        raise InvalidKeyLengthError if key&.length != self.class.expected_key_length
+        check_key_length
         encryptor.encrypt_and_sign contents
       end

@@ -108,5 +105,9 @@ def read_key_file
       def handle_missing_key
         raise MissingKeyError.new(key_path: key_path, env_key: env_key) if raise_if_missing_key
       end
+
+      def check_key_length
+        raise InvalidKeyLengthError if key&.length != self.class.expected_key_length
+      end
   end
 end
```